test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -120,10 +120,9 @@ TEST(os, test_random) {
   unsigned int seed = 1;
 
   // tty->print_cr("seed %ld for %ld repeats...", seed, reps);
-  os::init_random(seed);
   int num;
   for (int k = 0; k < reps; k++) {
-    num = os::random();
+    num = seed = os::next_random(seed);
```

Comment:
I suggest adding a comment before this line:
// Use next_random so the calculation is stateless.
Or something to that affect.

Reply:
Fixed.